fofix/core/Resource.py

```diff
@@ -229,6 +229,19 @@ def makeWritable(self, path):
         os.chmod(path, stat.S_IWRITE | stat.S_IREAD | stat.S_IEXEC)
 
     def load(self, target = None, name = None, function = lambda: None, synch = False, onLoad = None, onCancel = None):
+        """
+        Load a file into memory, as either a foreground or background operation.
+        
+        :param target: (unknown)
+        :param name: (unknown)
```

Comment:
I think it's the name of the resource

Reply:
If I understand you, then you are correct. I'm updating docs.